sample-apps/java-sample-app/cw-agent.json

```diff
@@ -0,0 +1,19 @@
+{
+  "agent": {
+    "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
```

Comment:
What are the references to `cw-agent` for?

Reply:
It is in the readme.

Reply:
I updated in the readme so that that it is explicit that this file is used for the purpose of pushing logs to cloudwatch.